src/PostInstall.php

```diff
@@ -0,0 +1,93 @@
+<?php
```

Comment:
Shouldn't this file be outside of the directory that will be used by the plugin?

With this current setup it will be loaded by the autoloader on the website, but it has no use there.